src/tests/cityCard.test.js

```diff
@@ -0,0 +1,22 @@
+import { render } from '@testing-library/react';
+import React from 'react';
+import { BrowserRouter, Routes, Route } from 'react-router-dom';
+import { Provider } from 'react-redux';
+import store from '../redux/store';
+import CityCard from '../components/CityCard';
+
+const fakeData = {
+  name: 'name', continent: 'continent', flag: 'flag', lat: 'lat', lon: 'lon',
+};
+
+test('should render CityCard', () => {
+  render(
+    <Provider store={store}>
+      <BrowserRouter>
+        <Routes>
+          <Route path="/" component={<CityCard props={fakeData} />} />
+        </Routes>
+      </BrowserRouter>
+    </Provider>,
+  );
+});
```

Comment:
- [ ] It would be better to add more tests for city cards and also add tests for the details page also using the `React testing library in order to match the requirements 
- screenshot of the requirements 
<img width="759" alt="Screenshot 2023-06-03 at 17 12 22" src="https://github.com/LDouglasOT/AirPollution/assets/61790587/b8a2f56f-ac7c-4239-a15d-021d86c0127c">